app/services/problem-service.js

```diff
@@ -38,8 +38,10 @@ define(['./module'], function(services) {
                     data: null
                 };
                 return $http(req);
+            },
+            getProblemPhotos: function(problemID) {
+                return $http.get('http://127.0.0.1:8000/api/problems/' + problemID + '/photos');
```

Comment:
it's time to put the string constant with api url to one separate place (so that you can change it in one place!). 
Please see Constants service in kv-007.LNU repository))